### PR TITLE
[15.0][ADD] hr_timesheet_day_week

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: hr_timesheet_day_week/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+{
+    "name": "Add Weekdays to time sheets",
```
Comment (dalonsod) [inline hr_timesheet_day_week/__manifest__.py]: Better this:
```suggestion
    "name": "Add day of week management to timesheets",
```
* 'Weekday' is different than 'day of week' AFAIK. 'Weekday' only refers to a day that is used to go work.
* 'Time sheet' is ambiguous, better official 'timesheet'.

File: hr_timesheet_day_week/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "description": "Add Weekdays to time sheets",
+    "category": "Timesheet",
+    "version": "15.0.1.0.0",
+    "depends": ["hr_timesheet"],
+    "data": [
+        "views/hr_timesheet_view.xml",
+    ],
+    "author": "Solvos Consultoría Informática",
+    "website": "https://github.com/solvosci/slv-timesheet",
+    "license": "AGPL-3",
```
Comment (dalonsod) [inline hr_timesheet_day_week/__manifest__.py]: Use LGPL by default, if no OCA module involved:
```suggestion
    "license": "LGPL-3",
```

File: hr_timesheet_day_week/models/account_analytic_line.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+    days_week = fields.Selection([
+        ("0", "Monday"),
+        ("1", "Tueday"),
+        ("2", "Weednesday"),
+        ("3", "Thurday"),
```
Comment (dalonsod) [inline hr_timesheet_day_week/models/account_analytic_line.py]: Typos:
```suggestion
        ("1", "Tuesday"),
        ("2", "Wednesday"),
        ("3", "Thursday"),
```

File: hr_timesheet_day_week/models/account_analytic_line.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+    days_week = fields.Selection([
```
Comment (dalonsod) [inline hr_timesheet_day_week/models/account_analytic_line.py]: IMO this is a better name:
```suggestion
    day_week = fields.Selection([
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only code review, see comments. Only cosmetic fixes or changes required, code looks to be good.

BTW, commit message should be "[ADD] hr_timesheet_day_week"

And, at last, why LICENSE and README.md root files are deleted?

File: hr_timesheet_day_week/models/account_analytic_line.py (hunk 3 above)
Comment (adriresu) [reply hr_timesheet_day_week/models/account_analytic_line.py]: Updated mentioned changes

**Comment by dalonsod** [comment]
Thanks, I'll test it ASAP

File: hr_timesheet_day_week/README.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Solvos ConsultorÃ­a InformÃ¡tica
```
Comment (dalonsod) [inline hr_timesheet_day_week/README.rst]: Please manually fix encoding typo

File: hr_timesheet_day_week/static/description/index.html [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+<a class="reference external" href="https://github.com/solvosci/slv_timesheet/issues/new?body=module:%20hr_timesheet_day_week%0Aversion:%2015.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**">feedback</a>.</p>
+<p>Do not contact contributors directly about support or help with technical issues.</p>
+</div>
+<div class="section" id="credits">
+<h1><a class="toc-backref" href="#id2">Credits</a></h1>
+<div class="section" id="authors">
+<h2><a class="toc-backref" href="#id3">Authors</a></h2>
+<ul class="simple">
+<li>Solvos ConsultorÃƒÂ­a InformÃƒÂ¡tica</li>
```
Comment (dalonsod) [inline hr_timesheet_day_week/static/description/index.html]: Please manually fix encoding typo

File: hr_timesheet_day_week/static/description/index.html [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+</ul>
+</div>
+<div class="section" id="contributors">
+<h2><a class="toc-backref" href="#id4">Contributors</a></h2>
+<ul class="simple">
+<li>AdriÃ¡n ResÃºa &lt;<a class="reference external" href="mailto:adrian.resua&#64;solvos.es">adrian.resua&#64;solvos.es</a>&gt;</li>
```
Comment (dalonsod) [inline hr_timesheet_day_week/static/description/index.html]: Please manually fix encoding typo

File: hr_timesheet_day_week/views/hr_timesheet_view.xml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        <field name="name">account.analytic.line.search</field>
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date']" position="after">
+                <field name="day_week"></field>
+            </xpath>
+            <xpath expr="//field[@name='date']" position="before">
+                <separator/>
+                <filter name="day_week" string="Weekend" domain="['|', ('day_week','=','5'), ('day_week','=','6')]"/>
```
Comment (dalonsod) [inline hr_timesheet_day_week/views/hr_timesheet_view.xml]: Cleaner code IMO:
```suggestion
                <filter name="day_week" string="Weekend" domain="[('day_week','in', ['5', '6'])]"/>
```

File: hr_timesheet_day_week/views/hr_timesheet_view.xml [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    <record id="view_account_analytic_line_days_week_tree_inherit" model="ir.ui.view">
+        <field name="name">account.analytic.line.tree</field>
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date']" position="before">
+                <field name="day_week" invisible="1"></field>
+            </xpath>
+            <xpath expr="//tree" position="attributes">
+                <attribute name="decoration-warning">day_week == '5' or day_week == '6'</attribute>
```
Comment (dalonsod) [inline hr_timesheet_day_week/views/hr_timesheet_view.xml]: Cleaner code:
```suggestion
                <attribute name="decoration-warning">day_week in ['5', '6']</attribute>
```

**Review by dalonsod — APPROVED** [review]
Locally tested, 👍 

Please only attend the later comments for cleaning code and encoding typos, and I'll merge it

**Comment by dalonsod** [comment]
Made some minor changes:

- Category in manifest.
- Weekend domain condition in views moved from `or` clauses to `in` clause.
- Some encoding typos in `README` and `index.html` files.

**Pull request merged**